### PR TITLE
Add notes for 2015-04-28

### PR DESCRIPTION
**Pull request by terrajobst**

### CODE REVIEW

**Comment by FiveTimesTheFun** [comment]
Hi Immo - the video link is still a 404, and all the links still have a timecode of 0:00

File: 2015-04-28-misc/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+* We shouldn't add an overload that takes a `CultureInfo` (due to layering). Instead, we should add it in the right place in glob
+* We shouldn't add the `char` overload as it's not very useful
+
+### #1378 Add IEnumerable\<KeyValuePair\<TKey, TValue>> variants to Dictionary's .ctors
+
+Status: **Accepting PRs** |
+[Issue](https://github.com/dotnet/corefx/issues/1378) |
+[Video](http://channel9.msdn.com/Blogs/dotnet/NET-Core-API-Review-2015-04-28#time=0h0m0s)
+
+* The implementation should throw if duplicate keys are detected (similar to what `Dictionary<K,V>` does when you pass in a collection with a different comparer when you an enumerable  an `ConcurrentDictionary<K,V>`).
```
Comment (davkean) [inline 2015-04-28-misc/README.md]: This sentence does not make grammatical sense.

File: 2015-04-28-misc/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* It seems we shouldn't expose the delta because that's not actually what people want, instead, they want the UTC offset
+
+However, the issue also asks about being able to construct the adjustment rules. That should be a separate issue because it's being a much bigger item:
+
+* They need to be able construct an `AdjustmentRule` and pass in the values that affects `BaseUtcOffsetDelta`
+* Currently, .NET Core doesn't support constructing `AdjustmentRule` at all
+
+### #1513 Add String.Split overloads that take a single char and string separator
+
+Status: **Not reviewed** |
```
Comment (davkean) [inline 2015-04-28-misc/README.md]: Should this be "*Accepting PRS"

Comment (terrajobst) [inline 2015-04-28-misc/README.md]: Good catch!

File: 2015-04-28-misc/README.md (hunk 1 above)
Comment (terrajobst) [inline 2015-04-28-misc/README.md]: Ooops. See, this is why we pay *you* the big bucks ;-)

**Pull request merged**